usersync/syncersbuilder.go

```diff
@@ -41,6 +43,7 @@ func BuildSyncers(hostConfig *config.Configuration, bidderInfos config.BidderInf
 			cfg.Key = bidder
 		}
 		cfgBySyncerKey[cfg.Key] = append(cfgBySyncerKey[cfg.Key], namedSyncerConfig{bidder, cfg})
+		bidderNameBySyncerKey[cfg.Key] = append(bidderNameBySyncerKey[cfg.Key], bidder)
```

Comment:
Can we re-use `cfgBySyncerKey` to also store `bidderName` ? both maps are using same keys and are passed to same function. 

Reusing `cfgBySyncerKey` will not increase number of input param in method signature. Also memory wise this will be less expensive compare to two maps with same keys holding different values

Reply:
Good point, Onkar! After exchange refactoring you see this kind of improvements right away :)
I briefly discussed possible way to pass a bidder name with Scott and it will be totally different. I will put a new PR for this.